tests/scanner/audit/data/bucket_retention_test_rules_0.yaml

```diff
@@ -0,0 +1,11 @@
+rules:
+  - name: exact retention 365
+    applies_to: 
+      - bucket
+    resource:
+      - type: organization
+        resource_ids:
+          - "433655558668"
+          - "433655558669"
+    minimum_retention: 365
+    maximum_retention: 365
```

Comment:
nit: add empty line at bottom of each file.

You can use the following vs code settings:

   "files.insertFinalNewline": true,
    "files.trimTrailingWhitespace": true,
    "files.trimFinalNewlines": true,